src/app/components/about/about.component.scss

```diff
@@ -19,14 +19,14 @@
   color: #4d4d4d;
   margin: 1.14rem 0 0.912rem 0;
   font-size: 34px;
-  font-weight: 500;
+  font-weight: 400;
```

Comment:
I see **400** font-weight used at other places as well. Can we use some variable like _$bolder_ / _$boldest_ etc. (depending on the other values of weight being used)

Also, at other places.

Reply:
yes we already have that one, will do that. Thanks :)

Reply:
@lunayach done